examples/single_model_scripts/run_openai_gpt.py

```diff
@@ -155,7 +155,8 @@ def main():
     special_tokens = ['_start_', '_delimiter_', '_classify_']
     tokenizer = OpenAIGPTTokenizer.from_pretrained(args.model_name, special_tokens=special_tokens)
```

Comment:
This line also won't work.

The way to add new (special) tokens in the tokenizer is now to use [`add_special_tokens`](https://huggingface.co/pytorch-transformers/main_classes/tokenizer.html#pytorch_transformers.PreTrainedTokenizer.add_special_tokens):
```
# Let's see how to add a new classification token to GPT-2
tokenizer = GPT2Tokenizer.from_pretrained('gpt2')
model = GPT2Model.from_pretrained('gpt2')

special_tokens_dict = {'cls_token': '<CLS>'}

num_added_toks = tokenizer.add_special_tokens(special_tokens_dict)
print('We have added', num_added_toks, 'tokens')
model.resize_token_embeddings(len(tokenizer))  # Notice: resize_token_embeddings expect to receive the full size of the new vocabulary, i.e. the length of the tokenizer.

assert tokenizer.cls_token == '<CLS>'
```

Reply:
I will have to double check, but this is the way I originally tried but it complained about unknown token being activated but not set.  Further, the tokens specified in the map dictionary weren't added for the GPT2 tokenizer.  Perhaps the  above were transient issues now rectified.  I did understand the above as the way the addition of additional special tokens were to be handled.